internal/commands/status.go

```diff
@@ -2,15 +2,30 @@ package commands
 
 import (
 	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+	"text/tabwriter"
+	"time"
 
 	"github.com/deislabs/duffle/pkg/action"
 	"github.com/deislabs/duffle/pkg/credentials"
 	"github.com/docker/app/internal"
+	"github.com/docker/app/internal/store"
 	"github.com/docker/cli/cli"
 	"github.com/docker/cli/cli/command"
+	units "github.com/docker/go-units"
 	"github.com/spf13/cobra"
 )
 
+var (
+	knownStatusActions = []string{
```

Comment:
So, the intention is to always prioritise `com.docker....status` over `io.cnab.status` rather than deprecate the former and try to move all bundles (including the ones app builds) to the latter over time?

I guess I'm unsure of the relationship between the two variants of status -- the code here suggests (by using `displayInstallationStatus` for both) that they are pretty similar?

Reply:
I think that needs some reflexion yes, in a follow-up. We should also think adding the support of the `json` status command, described in the [cnab spec](https://github.com/deislabs/cnab-spec/blob/master/804-well-known-custom-actions.md):
> io.cnab.status+json (with stateless: false and modifies: false): print a json payload describing the detailed status with the following schema (here is the json schema for this):